src/main/java/com/eu/habbo/messages/outgoing/room/engine/ObjectRemoveMultipleComposer.java

```diff
@@ -0,0 +1,27 @@
+package com.eu.habbo.messages.outgoing.room.engine;
+
+import com.eu.habbo.habbohotel.users.HabboItem;
+import com.eu.habbo.messages.ServerMessage;
+import com.eu.habbo.messages.outgoing.MessageComposer;
+import com.eu.habbo.messages.outgoing.Outgoing;
+import gnu.trove.set.hash.THashSet;
+
+public class ObjectRemoveMultipleComposer extends MessageComposer {
+
+    private final THashSet<? extends HabboItem> items;
+
+    public ObjectRemoveMultipleComposer(THashSet<? extends HabboItem> items) {
+        this.items = items;
+    }
+
+    @Override
+    protected ServerMessage composeInternal() {
+        this.response.init(Outgoing.ObjectRemoveMultiple);
+        this.response.appendInt(this.items.size());
+        for (HabboItem item : this.items) {
+            this.response.appendInt(item.getId());
+        }
+        this.response.appendInt(-1); // pickerId (-1 by default for AreaHiders)
```

Comment:
Can this also be not `-1`? Is this packet also used for `:pickall` on the real game?